### ISSUE TITLE
size of plot in browser

### ISSUE THREAD

**Issue by ghost** [timelyportfolio/rcdimple#26]
i run r markdown ioslides with a dimple chart and it plots perfectly in the preview, but not in the browser. 

this is perfect [rstudio]:
![preview rstudio](https://cloud.githubusercontent.com/assets/4662049/10316366/9e59e08a-6c67-11e5-8fdc-2c4a78ed2f5b.png).  

this is not [browser]: 
![selection_004](https://cloud.githubusercontent.com/assets/4662049/10316518/48c52d36-6c68-11e5-824f-2ee2f11a3733.png)

I have tried to set the plot size with set_bounds(x = 50, y = 50, width = 600, height = 450) [*]. This give the problem that the rollover label wraps to the wrong side, so you can't read what's there:

![selection_005](https://cloud.githubusercontent.com/assets/4662049/10316779/b7ef1ab8-6c69-11e5-9970-e2f266944404.png)

The data that I use are from a client, and I can't reproduce the problem with a stardard dataset like mtcars.

[*] the documentation terms this boundary setting sometimes as "setBounds" and sometimes as "set_bounds".

**Comment by timelyportfolio** [comment]
Are you setting an explicit `height` and `width` on the chart?

```
dimple(..., height = , width = )
```

I know there are quite a few issues/complications when combining ioslides with `htmlwidgets`, see https://github.com/ramnathv/htmlwidgets/issues/129.

**Comment by ghost** [comment]
yes, I have tried many combinations - with and without size, set_bounds. I don't have the problem with a normal markdown or in slidy. It seems to be related to ioslides.
